gpkit/constraints/tight.py

```diff
@@ -35,7 +35,7 @@ def process_result(self, result):
                         leftsubbed = constraint.left.sub(variables).value
                         rightsubbed = constraint.right.sub(variables).value
             if rel_diff >= self.reltol:
-                msg = ("Constraint [%.100s... %s %.100s] is not tight because "
+                msg = ("Constraint [%.100s... %s %.100s...] is not tight because "
```

Comment:
You need to add arguments to the tuple at the end of the string. `constraint` should be replaced by `constraint.left, constraint.oper, constraint.right`